<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>446</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>drawcli</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="viewportUpdateMode">
       <enum>QGraphicsView::FullViewportUpdate</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSave"/>
   <addaction name="actionSelection"/>
   <addaction name="actionLine"/>
   <addaction name="actionEllipse"/>
   <addaction name="actionRectangle"/>
   <addaction name="actionBringToFront"/>
   <addaction name="actionSendToBack"/>
   <addaction name="actionRotation"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionEllipse">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/circle.png</normaloff>:/image/circle.png</iconset>
   </property>
   <property name="text">
    <string>Ellipse</string>
   </property>
  </action>
  <action name="actionRectangle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/rectangle.png</normaloff>:/image/rectangle.png</iconset>
   </property>
   <property name="text">
    <string>Rectangle</string>
   </property>
  </action>
  <action name="actionSelection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/pointer.png</normaloff>:/image/pointer.png</iconset>
   </property>
   <property name="text">
    <string>Selection</string>
   </property>
  </action>
  <action name="actionBringToFront">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/sendtoback.png</normaloff>:/image/sendtoback.png</iconset>
   </property>
   <property name="text">
    <string>BringToFront</string>
   </property>
   <property name="toolTip">
    <string>BringToFront</string>
   </property>
  </action>
  <action name="actionSendToBack">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/bringtofront.png</normaloff>:/image/bringtofront.png</iconset>
   </property>
   <property name="text">
    <string>SendToBack</string>
   </property>
   <property name="toolTip">
    <string>sendToBack</string>
   </property>
  </action>
  <action name="actionRotation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/rotation.png</normaloff>:/image/rotation.png</iconset>
   </property>
   <property name="text">
    <string>rotation</string>
   </property>
   <property name="toolTip">
    <string>rotation</string>
   </property>
  </action>
  <action name="actionLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/linecolor.png</normaloff>:/image/linecolor.png</iconset>
   </property>
   <property name="text">
    <string>line</string>
   </property>
   <property name="toolTip">
    <string>line</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/save-5.png</normaloff>:/image/save-5.png</iconset>
   </property>
   <property name="text">
    <string>save</string>
   </property>
   <property name="toolTip">
    <string>save svg</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
